cmd/release/docs/main.go

```diff
@@ -1,183 +1,84 @@
 package main
 
 import (
-	. "./existing_docs"
-	. "./internal"
-	. "./new_docs"
-	. "./new_docs/templates"
-	"encoding/json"
-	"fmt"
-
 	"flag"
+	"fmt"
+	. "github.com/aws/eks-distro/cmd/release/docs/existing_docs"
+	. "github.com/aws/eks-distro/cmd/release/docs/new_docs"
+	"github.com/aws/eks-distro/cmd/release/utils"
+	. "github.com/aws/eks-distro/cmd/release/utils/git_manager"
 	"log"
+	"path/filepath"
 )
 
-const skipOverrideNumber = -1
-const verboseLogging=false
-
-/*
-Generate docs for release. Value for 'branch' flag must be provided.
-
-If a failure is encounter, attempts to undo any changes to files in branch doc directory, including deleting the
-directory if it was created.
-
-Caution about specific flags
-
-	usePrevReleaseManifestForComponentTable
-
-		If false, there if no guarantee that the generated table is correct for that release, as the generated table is
-		based on the previous release's release manifest (which must exist).
-
-		If true, the release manifest for that release must exist.
+const changeType = utils.Docs
 
-	isLocalReleaseNumberForNewRelease
+var rootDirectory = utils.GetGitRootDirectory()
 
-		If false, the local prod release number of the branch must be from the current release number, which will be
-		incremented for the next release. The generated docs are for the next release.
-
-		If true, the local prod release number of the branch must be the new release number. Typically, this means an
-		earlier PR incremented the release number, the PR was merged, and the change was pulled down locally. The
-		generated docs are for this release
-
-	overrideNumber and force
-
-		Both flags undercut the logical bedrock of the doc generation: the release number. The release number's impact
-		extends far beyond what is discernible by looking at the docs. Factors well outside the scope of this command,
-		such as the passage of time or code changes unrelated to the release, impact the generation of docs in
-		critically important ways that are not readily apparent.
-
-		Circumventing the expected and established workflow can easily result in errors that are not easily identifiable
```

Comment:
I love this warning, v serious

Reply:
yeah there's a reason why those flags were removed. They were a real "f*ck around and find out" situation